src/org/mirah/jvm/mirrors/better_scope.mirah

```diff
@@ -601,6 +652,14 @@ class MethodScope < BetterScope
     @locals = Locals.new
     @imports = ImportsAndSearchPackages.new
   end
+  
+  def initialize(source: ClosureScope, context: Node)
+    super(context)
+    @scoper     = source.internal_scoper
+    @locals     = source.internal_locals
+    @imports    = source.internal_imports
```

Comment:
I think these should defer to the source's, but not reuse them. Otherwise, if you have something like

```
foo do
  def f(a); end
  def g(a); end
end
```
wouldn't that cause an issue?